force_wfmanager/central_pane/analysis.py

```diff
@@ -1,9 +1,52 @@
 from pyface.tasks.api import TraitsTaskPane
-from traitsui.api import View
+
+from traits.api import List, Instance, Int, Any, Str, HasStrictTraits
+
+from traitsui.api import View, HGroup, VGroup, Spring
+
+
+class Result(HasStrictTraits):
+    name = Str()
+    value = Any()
+
+
+class EvaluationStep(HasStrictTraits):
+    _step = 0
+
+    @classmethod
+    def init_step_count(cls):
```

Comment:
I don't see the point of this. Just have an empty model. Actually, you don't even need the model at this stage. Just two empty panes.